web3_proxy/src/rpcs/many.rs

```diff
@@ -1119,9 +1173,18 @@ impl Web3Rpcs {
                         request_metadata.no_servers.fetch_add(1, Ordering::Release);
                     }
 
+                    // <<<<<<< HEAD
                     watch_consensus_rpcs.changed().await?;
 
                     watch_consensus_rpcs.borrow_and_update();
+                    // =======
+                    // TODO: i don't think this will ever happen
+                    // TODO: return a 502? if it does?
+                    // return Err(anyhow::anyhow!("no available rpcs!"));
+                    // TODO: sleep how long?
+                    // TODO: subscribe to something in ConsensusWeb3Rpcs instead
+                    sleep(Duration::from_millis(200)).await;
+                    // >>>>>>> 77df3fa (stats v2)
```

Comment:
delete this. we don't need to sleep anymore because we are awaiting watch_consensus_rpcs changing